### ISSUE TITLE
Dark theme causing eye strain

### ISSUE THREAD

**Issue by caugner** [mdn/fred#982]
### Discussed in https://github.com/mdn/yari/discussions/7704

<div type='discussions-op-text'>

<sup>Originally posted by **J-Cake** November 27, 2022</sup>
I'm lucky enough to be blessed with good eyesight, so generally I don't have an issue with taxing colour schemes or bright lights etc. but I can't help but notice that when ever I'm reading MDN with the dark theme, my eyes hurt after a few seconds, and I start to feel dizzy within a few minutes. I've never encountered this anywhere else so I assume it's related to the contrast or something technical (I have no idea about colour theory) but I thought I'd mention it. 

Perhaps lighten the blacks and dim the whites?

I do have to mention though, the light theme is 👌🏻 and the design scheme is pretty sleek. </div>

**Comment by caugner** [comment]
> With GitHub's dark mode, text has a contrast of 12.26 (13.30 in the editor), buttons have 9.86, and the headings at the top reach 15.89.
> 
> On MDN, the text has a contrast of [17.22](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=1B1B1B) (`--text-primary` on `--background-primary`), but the sidebars and the headings at the top only have [10.83](https://webaim.org/resources/contrastchecker/?fcolor=CDCDCD&bcolor=1B1B1B) (`--text-secondary` on `--background-primary`).

## Light theme

<img width="970" alt="image" src="https://user-images.githubusercontent.com/495429/204770191-a93713a6-6218-4f99-ad41-3a4c43c14123.png">

<img width="970" alt="image" src="https://user-images.githubusercontent.com/495429/204770261-ea4bbbc3-492e-4c09-b40c-e995e5d91334.png">

<img width="970" alt="image" src="https://user-images.githubusercontent.com/495429/204769823-28c08683-14fd-4c29-9180-d3f019b0d474.png">

<img width="970" alt="image" src="https://user-images.githubusercontent.com/495429/204769847-3a3c510f-52da-4fe4-8e62-0a0fccebec1d.png">

## Dark theme

<img width="970" alt="image" src="https://user-images.githubusercontent.com/495429/204769959-f9d1654a-a12d-4e6b-b59c-b7b0979ffe4d.png">

<img width="970" alt="image" src="https://user-images.githubusercontent.com/495429/204770005-9d02cd60-6830-44ff-b9ae-ac7d39e48467.png">

<img width="970" alt="image" src="https://user-images.githubusercontent.com/495429/204770058-e5779fc4-1fe8-4bae-9d99-b28f266ec161.png">

<img width="970" alt="image" src="https://user-images.githubusercontent.com/495429/204770097-e8f5cde6-dbbb-4b6f-9557-1c6a757c7d31.png">

**Comment by caugner** [comment]
@schalkneethling Chiming you in for your a11y expertise: Would it maybe make sense to have the current `--text-secondary` color as our `--text-primary` color and use the current `--text-primary` color only to highlight selected elements, such as the top nav (like GitHub does) and maybe the "Related Topics" and "In this article" headers in the sidebars, like we currently do?

According to WCAG, a contrast of 7:1 is recommended for text (see [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)), so 10.83 would still be a bit too contrasty, but already better.

**Comment by caugner** [comment]
I talked about this with @fiji-flo and since we don't have a high contrast theme (in contrast to GitHub), reducing the contrast makes our site less accessible for folks requiring high contrast. This means that this has to wait until our designer has time to take a look at this issue.

Thank you for your patience. 🙏 

PS: In the mean-time, you could use an extension like Dark Reader, which is also available for Firefox for Android.

**Comment by BerndHme** [comment]
I have the same issue.
Sometimes I go into the developer-tools  
... and change the color of the **--text-primary** to the same color as **--text-secondary**  
... way much better.

![image](https://github.com/mdn/fred/assets/132269491/bf8c0303-97c5-4dbe-b4f8-f46a97d2d167)
